build.sbt

```diff
@@ -49,7 +49,8 @@ val loggingDependencies = Seq(
 )
 
 val fileDependencies = Seq(
-  "com.github.pathikrit" %% "better-files" % "3.9.1"
+  "com.github.pathikrit" %% "better-files" % "3.9.1",
+  "com.lihaoyi" %% "os-lib" % "0.8.0"
```

Comment:
aren't both libraries for a single thing - working with files? maybe we can use one :)

Reply:
I will remove one